README.md

```diff
@@ -278,6 +278,26 @@ And on Windows by running:
 cmd /C "set NODE_ENV=development&&code"
 ```
 
+## Contributors
+
+Stylelint is maintained by volunteers. Without the code contributions from [all these fantastic people](https://github.com/stylelint/vscode-stylelint/graphs/contributors), Stylelint would not exist. [Become a contributor](CONTRIBUTING.md).
```

Comment:
[ask] `CONTRIBUTING.md` doesn't exist. Is there an alternative?

Reply:
```suggestion
Stylelint is maintained by volunteers. Without the code contributions from [all these fantastic people](https://github.com/stylelint/vscode-stylelint/graphs/contributors), Stylelint would not exist. [Become a contributor](https://github.com/stylelint/vscode-stylelint/contribute).
```

Good catch @ybiquitous, this works https://github.com/stylelint/vscode-stylelint/contribute and picks up the _shared_ `CONTRIBUTING.md` file from https://github.com/stylelint/.github/blob/main/.github/CONTRIBUTING.md

![image](https://user-images.githubusercontent.com/1016458/212624968-f3b9366d-434e-4b68-b366-03526e094586.png)

Reply:
Nice feature! I don't know it. LGTM 👍🏼